askomics/libaskomics/rdfdb/SparqlQueryBuilder.py

```diff
@@ -49,189 +65,116 @@ def get_neighbours_source_node(self, json, node_id):
         """
             get a node link with the id node
         """
-        list_name_node = []
-        for c in json['constraint']:
-            if c['link'] == 'node' and c['src'] == node_id:
-                list_name_node.append(c['tag'])
-
-        list_nodes = []
-        for n in list_name_node:
-            list_nodes.append(self.get_node(json, n))
-
-        return list_nodes
-
-    def more_than_one_child_with_the_same_uri(self, lneib):
-        for c in lneib:
-            for d in lneib:
-                if c['uri'] == d['uri'] and c['id'] != d['id']:
-                    return True
-        return False
-
-    def build_new_json_by_unionpath(self, json, node_id, lneib): # FIXME lneib is unused
```

Comment:
Can you keep it (and add a FIXME deadcode) as it is used by the dead get_block_request method?